google/cloud/security/common/util/metadata_server.py

```diff
@@ -20,29 +20,87 @@
 import httplib
 import socket
 
+
+METADATA_SERVER_HOSTNAME = 'metadata.google.internal'
 METADATA_SERVER_CONN_TIMEOUT = 2
+REQUIRED_METADATA_HEADER = {'Metadata-Flavor': 'Google'}
+HTTP_SUCCESS = httplib.OK
+HTTP_GET = 'GET'
+
+
+class MetadataBaseError(Exception):
+    """Base error class for handling meta_data errors."""
```

Comment:
Could you put the errors in a common util_errors module or similar? Also, we are following the following conventions for custom error classes:

1. Have a base error class that inherits from Exception (e.g. https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/gcp_api/errors.py#L18)
2. Custom error classes inherit from the base error class